### PR TITLE
SG Balance adjustments

### PR DESCRIPTION
**Pull request by Shad0vvs**
## Why It's Good For The Game

Bugfix + needed balance change as SG's are too strong at the moment.

## Changelog

:cl:
balance: Smartgun firerate adjusted to ~400 RPM, Full auto scatter peak set to 80 shots, Scatter amount set to 8.
fix: Fixed WO Terminator's dirty smartgun flags.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/smartgun.dm [hunk 1]
```diff
@@ -611,6 +611,9 @@
 	if(!recoil_compensation)
 		scatter = SCATTER_AMOUNT_TIER_8
 	burst_scatter_mult = SCATTER_AMOUNT_TIER_10
+	fa_delay = FIRE_DELAY_TIER_10
+	fa_scatter_peak = FULL_AUTO_SCATTER_PEAK_TIER_10
+	fa_max_scatter = SCATTER_AMOUNT_NONE
```
Comment (stanalbatross) [inline code/modules/projectiles/guns/smartgun.dm:616]: why

File: code/__DEFINES/weapon_stats.dm [hunk 2]
```diff
@@ -145,6 +145,7 @@ As such, don't expect any values assigned to common firearms to even consider ho
 #define FIRE_DELAY_TIER_7 	4
 #define FIRE_DELAY_TIER_8 	3
 #define FIRE_DELAY_TIER_9 	2
+#define FIRE_DELAY_TIER_SG 	1.5
 #define FIRE_DELAY_TIER_10 	1
 #define FIRE_DELAY_TIER_SMG	1.3
```
Comment (stanalbatross) [inline code/__DEFINES/weapon_stats.dm:150]:
```suggestion
#define FIRE_DELAY_TIER_10 	1
#define FIRE_DELAY_TIER_SG 	1.5
#define FIRE_DELAY_TIER_SMG	1.3
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/smartgun.dm (hunk 1 above)
Comment (Shad0vvs) [reply code/modules/projectiles/guns/smartgun.dm:616]: You wanted the terminator ones to frag?

**Review by stanalbatross — APPROVED** [review]

**Comment by stanalbatross** [comment]
this should be a nice middleground + not too strong

**Pull request merged**